governance/proposals/config.js

```diff
@@ -0,0 +1,11 @@
+const ARB_TOKEN_ADRESS_ON_L2 = '0x912CE59144191C1204E64559FE8253a0e49E6548' // ARB TOKEN ADDRESS ON ARBITRUM ONE
+const GRANTS_CONTRACT_ADDRESS = '0x00D5E0d31d37cc13C645D86410aB4cB7Cb428ccA'
+const TIMELOCK_L2_ALIAS = '0x28ffDfB0A6e6E06E95B3A1f928Dc4024240bD87c' // Timelock Alias Address on L2
+const L1_TIMELOCK_CONTRACT = '0x17EEDFb0a6E6e06E95B3A1F928dc4024240BC76B' // Timelock Address mainnet
```

Comment:
Same here I am pretty sure it is already defined elsewhere. Please use that!

Reply:
I'm not sure what exactly you are referring to here, please clarify?
For ```L1_TIMELOCK_CONTRACT```, I noticed it has been defined and used in several other files but I could not find it exported in any of them, to be used here.